helm/files/bootstrap.py

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/env python
+import http.client
+import json
+import os
+import ssl
+import subprocess
+import sys
+from pprint import pprint
+from urllib.parse import urlencode
+
+subprocess.check_call([sys.executable, "-m", "pip", "install", "kubernetes"])
+
+import kubernetes.client, kubernetes.config
+from kubernetes.client.rest import ApiException
+
+NAMESPACE = os.environ.get("KUBERNETES_NAMESPACE", "ns1")
+HOST = os.environ.get("BOOTSTRAP_HOST", "core")
+PORT = os.environ.get("BOOTSTRAP_PORT", 80)
+BOOTSTRAP_USER = os.environ.get("BOOTSTRAP_USER", "nsone")
+BOOTSTRAP_PASSWORD = os.environ.get("BOOTSTRAP_PASSWORD", "secure-password")
+FIRST_USER = os.environ.get("FIRST_USER", "ns1")
+FIRST_USER_PASSWORD = os.environ.get("FIRST_USER_PASSWORD", "privatedns")
+
+
+def request(method, domain, path="/", **kwargs):
+    if domain.startswith("https://"):
+        ctx = ssl.create_default_context()
+        ctx.check_hostname = False
+        ctx.verify_mode = ssl.CERT_NONE
+        conn = http.client.HTTPSConnection(domain[8:], PORT, context=ctx)
+    else:
+        conn = http.client.HTTPConnection(domain, PORT)
+
+    if isinstance(kwargs, dict) and kwargs.get("params"):
```

Comment:
I don't think `kwargs` can be anything but a dictionary. In which case, no need to check type here.

Reply:
This is a bit nitpicky, so no requirement to make a change.  If you're not going to use the result from the `get`, then you can just use `in` to check membership.

Alternatively, make the get call above:
```
params = kwargs.get("params")
```
then check after
```
if params is not None:
```

Functionally, I think it makes little difference.

Reply:
@cbertinato -- I copied this whole file from the bootstrap script in platform with slight modifications for k8s. I'm wondering if this comment and your others on this file should be addressed in that repo and then carried over to here? I'm not opposed to making them here, but I'd rather keep this script as similar as possible to the one in platform

Reply:
Ah, didn't realize that. Then pay no mind. 😄Probably better to address these there.